sdk/computefleet/azure-resourcemanager-computefleet/src/test/java/com/azure/resourcemanager/computefleet/ComputeFleetManagerTests.java

```diff
@@ -0,0 +1,276 @@
+            .authenticate(credential, profile);
+
+        networkManager = NetworkManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @LiveOnly
+    public void testCreateComputeFleet() {
+        Fleet fleet = null;
+        try {
+            String fleetName = "fleet" + randomPadding();
+            String vmName = "vm" + randomPadding();
+            String vnetName = "vnet" + randomPadding();
+            String loadBalancerName = "loadBalancer" + randomPadding();
+            String publicIpName = "publicIp" + randomPadding();
+            String adminUser = "adminUser" + randomPadding();
+            String adminPwd = UUID.randomUUID().toString().replace("-", "@").substring(0, 13);
+            // @embedStart
+            Network network = networkManager.networks()
+                .define(vnetName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withAddressSpace("172.16.0.0/16")
+                .defineSubnet("default")
+                .withAddressPrefix("172.16.0.0/24")
+                .attach()
+                .create();
+
+            PublicIpAddress publicIpAddress = networkManager.publicIpAddresses()
+                .define(publicIpName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withSku(PublicIPSkuType.STANDARD)
+                .withIpAddressVersion(IpVersion.IPV4)
+                .withStaticIP()
+                .create();
+
+            LoadBalancer loadBalancer = networkManager.loadBalancers()
+                .define(loadBalancerName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .defineLoadBalancingRule(loadBalancerName + "-lbrule")
+                .withProtocol(TransportProtocol.TCP)
+                .fromExistingPublicIPAddress(publicIpAddress)
```

Comment:
Same issue, can it work with LoadBalancer without public IPaddress? E.g. `.fromExistingSubnet`?

We don't want to have any public IP, if doable.

Reply:
Fixed in the new version.